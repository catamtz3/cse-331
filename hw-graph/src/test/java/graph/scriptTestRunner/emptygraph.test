# Create an empty graph
CreateGraph emptyGraph

# Get the nodes of the graph
ListNodes emptyGraph

