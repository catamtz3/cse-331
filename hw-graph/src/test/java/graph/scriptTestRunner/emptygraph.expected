# Create an empty graph
Created graph emptyGraph

# Get the nodes of the graph
emptyGraph contains:
